src/defaultSettings.js

```diff
@@ -20,6 +20,8 @@ const defaultSettings = {
     boxplots: true,
     violins: false,
     missingValues: ['', 'NA', 'N/A'],
+    unscheduled_visits: false,
+    unscheduled_visit_pattern: /unscheduled|early termination/i,
```

Comment:
I would probably lean towards making this text based instead of requiring users to understand regex. Or I suppose we could support both by adding `unscheduled_visit_terms:["unscheduled","early termination"]` alongside the `unscheduled_visit_pattern` and then building some logic regarding which one takes precedence ...

Reply:
The problem is unscheduled visits typically have a suffix that indicates where the visit fell in the schedule of events.  I've added another parameter, `unscheduled_visit_values` that default to `null` but overrides the `unscheduled_visit_pattern` if specified.